pkg/volume/cinder/volumeservice/connection.go

```diff
@@ -39,7 +39,24 @@ import (
 )
 
 type cinderConfig struct {
-	openstack_provider.Config
```

Comment:
I had completely forgotten that part of this code was duplicated in the drivers. :(

Reply:
should this also be clientconfigified?